optimism/src/keccak/interpreter.rs

```diff
@@ -130,6 +136,8 @@ pub trait KeccakInterpreter<F: One + Debug + Zero> {
     /// So:
     /// - At most, 474 constraints are added per row
     fn constraints(&mut self) {
+        self.check_selectors();
```

Comment:
How is `check_selectors` related to `checks`? And how are these related to `check`? It seems that they are all very similar in their implementation with quite generic naming.

Reply:
`check_selectors` are no longer used in later commits. There will only be `check` to check one thing, and `checks` which adds all the checks that could be relevant for the witness but which do not require constraints for them. Basically, just to check that I am configuring the selectors correctly in the witness side.